components/tests/ui/testcases/web/webadmin_create_group_and_user.txt

```diff
@@ -135,7 +132,6 @@ Create Edit User
 
     Input Text                  first_name    ${user_name}-Edited
     Click Button                Save
-    Location Should Be          ${USERS URL}
```

Comment:
Was this failing due to a timeout?
It would be nice to check that users are returned to the correct page.
Maybe move it after the following keyword ``` Wait Until Page Contains    ${user_name}-Edited ``` or use
```Wait Until Keyword Succeeds    Location Should Be          ${USERS URL}```

Same for test above.

Reply:
I don't think it had the expected effect anyway
It works in chrome but failed on firefox on CI master
I will roll it back

Reply:
I will try the suggestion